ddtrace/contrib/pymongo/patch.py

```diff
@@ -1,14 +1,49 @@
 import pymongo
 
+from ddtrace import Pin
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+
+from ...ext import mongo as mongox
+from ...utils.deprecation import deprecated
 from .client import TracedMongoClient
 
 # Original Client class
 _MongoClient = pymongo.MongoClient
 
-
 def patch():
+    # We should progressively get rid of TracedMongoClient. We now try to
+    # wrap methods individually. cf issue #_
```

Comment:
Shall I create an issue to migrate pymongo integration from ObjectProxy to method wrappers?

Reply:
Sure! That issue would cover essentially rewriting the pymongo integration. Let's treat these as a separate issues to keep things simple 🙂.